sdk/servicebus/service-bus/README.md

```diff
@@ -13,7 +13,7 @@ Use the client library for Azure Service Bus in your Node.js application to
 
 [Source code for v1.1.5](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/service-bus_1.1.5/sdk/servicebus/service-bus) |
 [Package for v1.1.5 (npm)](https://www.npmjs.com/package/@azure/service-bus/v/1.1.5) |
-[API Reference Documentation for v1.1.5](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/1.1.5/index.html) |
+[API Reference Documentation for v1.1.5](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/?view=azure-node-latest) |
```

Comment:
This link currently shows older version of docs, it'll take a couple more days to reflect what's on.. 
https://review.docs.microsoft.com/en-us/javascript/api/@azure/service-bus/?view=azure-node-latest&branch=smoke-test